.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  tests:
+    name: Python ${{ matrix.python-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
```

Comment:
Should we add `fail-fast: false`?

Reply:
I don't have a problem with the default fail fast. You do you though!